client/src/components/main/side.nav.tsx

```diff
@@ -1,116 +1,126 @@
-          </Link>
-          <Link>
-            <h4>Favorites</h4>
-          </Link>
-          <div className="wrapper-nav">
-            <div className="scrollable">
-              <Accordion defaultIndex={[0]} allowMultiple>
-                <AccordionItem>
-                  <h2>
-                    <AccordionButton>
-                      <Link>
-                        <h4>My circle</h4>
-                      </Link>
-                      <AccordionIcon />
-                    </AccordionButton>
-                  </h2>
-
-                  {user?.user.circle.map((elm) => {
-                    return (
-                      <AccordionPanel pb={4}>
-                        <Link>
-                          <button
-                            onClick={() => {
-                              navigate(`/app/person-details/${elm._id}`);
-                            }}
-                          >
-                            <h4>{elm.name}</h4>
-                          </button>
-                        </Link>
-                      </AccordionPanel>
-                    );
-                  })}
-                </AccordionItem>
-              </Accordion>
-            </div>
-          </div>
-          <Link
-            onClick={() => {
-              navigate(`/app/add-person/${userId}`);
-            }}
-          >
-            <h4>Add to my circle</h4>
-          </Link>
-        </div>
-        <div>
-          <Link>
-            <h4>Help</h4>
-          </Link>
-          <Link
-            onClick={() => {
-              handleLogout();
-              navigate("/login");
-            }}
-          >
-            <h4>Logout</h4>
-          </Link>
-        </div>
-      </nav>
-    </>
-  );
+
+
+    return (
+        <>
+            <nav className="column-items">
+                <div>
+                    <Link href={`/app/dashboard/${user?.user._id}`}>
+                        <h4>Dashboard</h4>
+                    </Link>
+
+                    <Link href={`/app/favorites/${user?.user._id}`}>
+                        <h4>Favorites</h4>
+                    </Link>
+
+                    <div className="wrapper-nav">
+                        <div className="scrollable">
+                            <Accordion defaultIndex={[0]} allowMultiple>
+                                <AccordionItem>
+                                    <h2>
+                                        <AccordionButton>
+                                            <Link>
+                                                <h4>My circle</h4>
+                                            </Link>
+                                            <AccordionIcon />
+                                        </AccordionButton>
+                                    </h2>
+
+                                    {user?.user.circle.map((elm) => {
+                                        return (
+                                            <AccordionPanel pb={4}>
```

Comment:
If you map in react, make sure each child gets a `key`:

<AccordionPanel key={elm.id}> for example if elm has an id